src/applications/caregivers/tests/components/ConfirmationPage/ConfirmationPrintView.unit.spec.js

```diff
@@ -1,33 +1,53 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import { expect } from 'chai';
-import moment from 'moment';
 
 import ConfirmationPrintView from '../../../components/ConfirmationPage/ConfirmationPrintView';
 
-describe('hca <ConfirmationScreenView>', () => {
-  it('should render Confirmation Screen View with name and timestamp', () => {
-    const name = {
+describe('CG <ConfirmationPrintView>', () => {
+  const defaultProps = {
+    name: {
       first: 'John',
       middle: 'Marjorie',
       last: 'Smith',
       suffix: 'Sr.',
-    };
+    },
+    timestamp: undefined,
+  };
 
-    const timestamp = 1666887649663;
+  it('should render with default props', () => {
+    const view = render(<ConfirmationPrintView {...defaultProps} />);
+    const selectors = {
+      image: view.container.querySelector('.vagov-logo'),
+      title: view.container.querySelector('h1'),
+      subtitles: view.container.querySelectorAll('h2'),
+      veteranName: view.container.querySelector(
+        '[data-testid="cg-veteranfullname"]',
+      ),
+    };
+    expect(selectors.image).to.exist;
+    expect(selectors.title).to.contain.text(
+      'Apply for the Program of Comprehensive Assistance for Family Caregivers',
```

Comment:
Hey Matt what are you thoughts on maybe creating a Constants file that houses all the copy for easier management in case copy changes in the future and breaks tests?

Reply:
This has been updated and moved to the content file that we do have existing. We do have a greater initiative that has been tossed around about offering the form in multiple languages, which would cause us to take all content and move it to a constants file, so that is definitely on the radar, too! 🚀